### PR TITLE
c-chain to eth address transfer

### PR DESCRIPTION
**Pull request by blackwidoq**

### CODE REVIEW

File: network-documentation/avalanche/tutorials/AVAX_C_chain_to_ETH_address_transfer_tutorial.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ description: Teaching how to Transfer AVAX native tokens from C chain to ETH add
 
 ## About the author
 
-This tutorial was created by [Seongwoo Oh](https://github.com/blackwidoq). He is a student and an Avalanche enthusiast. 
+This tutorial was created by [Seongwoo Oh](https://github.com/blackwidoq). He is a student and an Avalanche novice.
```
Comment (pyr0gan) [inline network-documentation/avalanche/tutorials/AVAX_C_chain_to_ETH_address_transfer_tutorial.md:7]: Movin' on up, hey? 😁

**Review by pyr0gan — APPROVED** [review]

File: network-documentation/avalanche/tutorials/AVAX_C_chain_to_ETH_address_transfer_tutorial.md (hunk 1 above)
Comment (blackwidoq) [reply network-documentation/avalanche/tutorials/AVAX_C_chain_to_ETH_address_transfer_tutorial.md:7]: thought i need to be more honest about where i am haha.  I am going to have to ask my friend to show me how to resolve complex conflicts. I will have this resolved later today. thank you

**Review by pyr0gan — CHANGES_REQUESTED** [review]
Please resolve all conflicts. It might be easier to continue using the existing branch you submitted the tutorial on. It appears from the changes that you had a few conflicts while trying to edit. How are you editing your markdown, if I may ask?

**Comment by blackwidoq** [comment]
@pyr0gan   I dont actually really know how to view what's causing the conflict. I explored using `git mergetool` but havent been successful.  I will ask my friend who is very good with this stuff later today and get it resolved.  Thank you!

**Review by pyr0gan — APPROVED** [review]

**Pull request merged**